src/main/java/request/HttpRequest.java

```diff
@@ -0,0 +1,84 @@
+package request;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import util.IOUtils;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.StringTokenizer;
+
+public class HttpRequest {
+    private String method;
+    private String url;
+    private String httpVersion;
+    private Map<String, String> params = new HashMap<>();
+    private final Logger log = LoggerFactory.getLogger(getClass());
+
+    public void init(String requestLine) {
```

Comment:
init 보다 조금 더 구체적인 이름이 좋을 것 같습니다.
```suggestion
    public void setRequestLine(String requestLine) {
```